forms-flow-api/src/formsflow_api/resources/cmis_connector.py

```diff
@@ -39,15 +39,22 @@ def post():
             uploads = repo.getObjectByPath("/uploads")
 
         if 'upload' not in request.files:
-            return {"message": "No upload files in the request"}, HTTPStatus.INTERNAL_SERVER_ERROR
+            return {"message": ""
+                               "No upload files in the request"}, HTTPStatus.INTERNAL_SERVER_ERROR
```

Comment:
@shibin-aot pls check the response format. Also, we can use the status codes in the 400 range for the client error like BAD_REQUEST